web/views/color_view.ex

```diff
@@ -2,5 +2,8 @@ defmodule Streamr.ColorView do
   use Streamr.Web, :view
   use JaSerializer.PhoenixView
 
-  attributes [:normal, :deuteranopia, :tritanopia, :protanopia, :order]
+  attributes [:hex, :normal, :deuteranopia, :tritanopia, :protanopia]
```

Comment:
Yep! It serializes the color that should be used for the frontend for the current user. It default `normal`. 

The other attributes are there so users can optionally see available color palettes on the frontend.